data/categories.txt

```diff
@@ -785,7 +785,7 @@ shop-car
 en:car shop|U+1F697|U+1F698|U+1F699
 ru:автосалон|магазин
 uk:автосалон|магазин
-de:Auto-Shop|Laden
+de:Autohaus
```

Comment:
Ok. There should be some documentation somewhere about how this file is processed. My lack of knowledge also resulted in removing some of the "Sehenswürdigkeit" below.   
Hint: "Geschäft" would IMHO be a better category name then "Laden"